packages/uiweb/src/lib/components/space/SpaceWidget/SpaceWidget.tsx

```diff
@@ -5,30 +5,26 @@ import { SpaceDTO } from '@pushprotocol/restapi';
 import { WidgetContent } from './WidgetContent';
 import { WidgetHeader } from './WidgetHeader';
 
-import { ENV } from '../../../config';
 import { ISpaceWidgetProps } from '../exportedTypes';
-
-export interface ISpaceWidgetComponentProps extends ISpaceWidgetProps{
-  env: ENV;
-}
+import { useSpaceData } from '../../../hooks';
 
 const DEFAULT_OFFSET = 16;
 const DEFAULT_MAXWIDTH = 415;
 
-export const SpaceWidget: React.FC<ISpaceWidgetComponentProps> = (options: ISpaceWidgetComponentProps) => {
+export const SpaceWidget: React.FC<ISpaceWidgetProps> = (options: ISpaceWidgetProps) => {
   const {
     bottomOffset = DEFAULT_OFFSET,
     rightOffset = DEFAULT_OFFSET,
     width,
     zIndex = 1000,
-    env, 
     spaceId,
     shareUrl,
     onClose = (() => { /** */}) as MouseEventHandler<HTMLDivElement>,
 
     isHost, isLive, isJoined, isMember, isTimeToStartSpace
   } = options || {};
   const [widgetHidden, setWidgetHidden] = useState(!spaceId);
+  const { env } = useSpaceData();
```

Comment:
place it after the useState hooks

ref: https://www.notion.so/pushprotocol/Frontend-Coding-Practices-React-JavaScript-TypeScript-8484db76d828435eb847b3856f42efa1